__tests__/app.test.js

```diff
@@ -0,0 +1,17 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection");
+
+afterAll(() => db.end());
+
+describe("1: GET /api/reviews", () => {
+	test("The url responds with a status of 200", () => {
+		return request(app)
+			.get("/api/reviews")
+			.expect(200)
+			.then(({ body }) => {
+				const { reviews } = body;
+				expect(reviews).toBeInstanceOf(Array);
+			});
```

Comment:
This test is a little lacking - yes we want to assert that an array of objects is on the response body but crucially we need to assert what they look like - also see main comment to PR review

Reply:
Modified